dev/io.openliberty.restfulWS.3.0.cdi.3.0_fat/test-applications/basic/src/io/openliberty/restfulWS30/cdi30/fat/basic/CommentError.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 21 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2021 IBM Corporation and others.
```

Ah, the year 21 AD - I remember it like it was yesterday, but I was pretty young back then...  :-)